### ISSUE TITLE
Show line number for current file mode

### ISSUE THREAD

**Issue by Arcanemagus** [facebook-atom/atom-ide-ui#101]
### Description
The line number of a `diagnostic` when in `Current File Only` mode is impossible to determine from the table... or to sort by.

### Expected Behavior
The line number is shown as a column in `Current File Only` mode.

### Actual Behavior
The line number isn't shown, meaning if any other column is accidentally clicked the messages are sorted in a near useless manner with no way to get back to a usable list without disabling `Current File Only` mode.

### Versions
* Atom: v1.22.0-beta1
* Client OS: Windows 10 x64 Enterprise
* atom-ide-ui: v0.5.2

### Additional Details
<details>
<summary>Installed packages</summary>

```
Community Packages (77) C:\Users\username\.atom\packages
├── atom-beautify@0.30.5
├── atom-ide-ui@0.5.2
├── atom-material-ui@2.0.6
├── autocomplete-lua@0.9.0
├── busy-signal@1.4.3 (disabled)
├── docblockr@0.11.0
├── editorconfig@2.2.2
├── file-watcher@1.2.6
├── highlight-selected@0.13.1
├── ide-flowtype@0.17.4
├── ide-php@0.6.9
├── ide-python@0.2.2
├── ide-typescript@0.6.2
├── intentions@1.1.5 (disabled)
├── language-ansible@0.2.1
├── language-chef@0.11.0
├── language-cjson@0.0.1
├── language-docker@1.1.8
├── language-elixir@0.20.3
├── language-erlang@3.2.0
├── language-fortran@2.1.6
├── language-gettext@0.6.1
├── language-glsl@2.0.2
├── language-haml@0.25.2
├── language-haskell@1.14.3
├── language-icinga2@0.3.0
├── language-ini@1.19.0
├── language-jade@0.7.2
├── language-jolie@0.5.0
├── language-kotlin@0.5.0
├── language-livescript@0.0.3
├── language-lua@0.9.11
├── language-moonscript@1.7.1
├── language-postcss@1.3.1
├── language-powershell@4.0.0
├── language-puppet@0.23.0
├── language-r@0.4.2
├── language-reg@0.0.0
├── language-rust@0.4.12
├── language-swift@0.5.0
├── language-terraform@0.8.1
├── language-vue@0.23.1
├── linter@2.2.0
├── linter-coffeelint@1.3.1
├── linter-eslint@8.4.0
├── linter-flake8@2.2.1
├── linter-htmlhint@1.3.4
├── linter-js-standard@4.0.2
├── linter-js-yaml@1.2.8
├── linter-jsonlint@1.3.0
├── linter-markdown@5.2.0
├── linter-perl@0.8.1
├── linter-perlcritic@2.0.0
├── linter-php@1.5.1
├── linter-phpcs@1.6.8
├── linter-phpmd@2.0.0
├── linter-proselint@3.3.0
├── linter-pylama@0.9.4
├── linter-pylint@2.1.0
├── linter-rails-best-practices@0.2.2 (disabled)
├── linter-reek@2.2.2
├── linter-rubocop@2.2.0
├── linter-ruby@1.2.6
├── linter-stylelint@4.0.2
├── linter-tidy@2.3.1
├── linter-ui-default@1.6.10
├── linter-ui-plus@0.3.2
├── mapfile-grammar@0.1.0
├── minimap@4.29.7
├── minimap-git-diff@4.3.1
├── minimap-highlight-selected@4.6.1
├── minimap-linter@2.1.1
├── minimap-selection@4.5.0
├── sequential-number@0.5.0
├── sort-lines@0.18.0
├── toggle-quotes@1.0.1
└── trailing-spaces@0.4.0
```
</details>

**Comment by matthewwithanm** [comment]
Yeah, sorry about this. We have an internal issue tracking it too. Should get to it next week (:

**Comment by hansonw** [comment]
Fixed with 306e20999337208d2800982bf21f6817380f1e48, I'll put out a release after some other fixes are checked in (#105 is also really pesky)
